shared/include/userver/utils/fixed_array.hpp

```diff
@@ -89,6 +114,28 @@ FixedArray<T>::FixedArray(std::size_t size, Args&&... args)
   }
 }
 
+template <class T>
+template <class Range>
+FixedArray<T>::FixedArray(FromRangeTag /*tag*/, Range&& range)
+    : storage_(nullptr), size_(std::size(std::as_const(range))) {
+  storage_ = std::allocator<T>{}.allocate(size_);
+
+  auto* begin = data();
+  auto* const end = begin + size_;
+  auto their_begin = range.begin();
+
+  try {
+    for (; begin != end; ++begin) {
+      new (begin) T(*their_begin);
```

Comment:
Move elements if Range is an rvalue

Reply:
No, thanks. What about `std::span<T>&&`? Standard uses `move_iterator` for such purposes